modules/client/examples/06-multisig-client/04-approve-proposal.ts

```diff
@@ -0,0 +1,38 @@
+/* MARKDOWN
+### Approve a multisig proposal
+*/
+import {
+  Context,
+  ContextPlugin,
+  MultisigClient,
+  ApproveProposalSteps,
+} from "@aragon/sdk-client";
+import { ApproveMultisigProposalParams } from "../../src";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+
+const voteParams: ApproveMultisigProposalParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890",
+  proposalId: "0x1234567890123456789012345678901234567890",
+};
+
+const steps = client.methods.approveProposal(voteParams);
```

Comment:
We also need another operation: `execute()`. In case someone decides not to pay the execution costs him/herself